internal/proto/controller/api/services/v1/worker_service.proto

```diff
@@ -123,6 +123,26 @@ service WorkerService {
       summary: "Removes api tags from an existing Worker."
     };
   }
+
+  // ReadCertificateAuthority returns the current and next set of root certificates
+  rpc ReadCertificateAuthority(ReadCertificateAuthorityRequest) returns (ReadCertificateAuthorityResponse) {
+    option (google.api.http) = {
+      get: "/v1/workers:read-cas"
```

Comment:
While some equate one root == one certificate authority in my mind the authority is the thing responsible for the roots and using them. So I think this would be singular -- `read-ca` and `reinitialize-ca`.

Also there is a mismatch here between the action on the API and the action in the grants, which are `read-certificate-authority` and `reinitialize-certificate-authority`, that needs to be solved since the two need to match. Personally I lean towards the latter, we have tended to favor explicitness over brevity, but I could be convinced either way. Maybe poll the BE channel?